pvm/ts/src/state/interfaces.ts

```
@@ -33,11 +33,12 @@ export interface IGameManagement {
     getAll(): Promise<IGameStateDocument[]>;
     get(address: string): Promise<any | null>;
     create(nonce: bigint, contractSchemaAddress: string, gameOptions: GameOptions): Promise<string>;
-    save(sate: IJSONModel): Promise<void>;
     saveFromJSON(json: any): Promise<void>;
 }
 
 export interface ITransactionManagement {
+    addTransaction(tx: Transaction): Promise<void>;
+    addTransactions(txs: Transaction[], blockHash: string): Promise<void>;
     exists(txid: string): Promise<Boolean>;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)